test/environmentSettings.ts

```diff
@@ -120,21 +125,34 @@ export class EnvironmentSettings {
     return this._javaHome;
   }
 
-  public get useExistingProject(): string | undefined {
-    return this._useExistingProject;
+  public get localProjectPath(): string | undefined {
+    return this._localProjectPath;
   }
 
-  public set useExistingProject(existingProject: string | undefined) {
-    const projectPath = existingProject ?? process.env.USE_EXISTING_PROJECT_PATH;
+  public get githubProjectUrl(): string | undefined {
+    return this._githubProjectUrl;
+  }
+
+  public set localProjectPath(localPath: string | undefined) {
+    const projectPath = localPath ?? process.env.LOCAL_PROJECT_PATH;
     if (!projectPath) {
-      this._useExistingProject = undefined;
+      this._localProjectPath = undefined;
       return;
     }
     if (!fs.existsSync(projectPath)) {
       throw new Error(`Project path for "${projectPath}" does not exist`);
     }
+    this._localProjectPath = projectPath;
+  }
+
+  public set githubProjectUrl(githubUrl: string | undefined) {
+    const projectPath = githubUrl ?? process.env.GITHUB_PROJECT_URL;
```

Comment:
Would you mind changing the local var name projectPath to something more suitable for handling URL?